src/containers/AppStore/containers/AppStoreHome/AppStoreHero/Styles.ts

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
+import {Container as SharedContainer} from 'components';
+import colors from 'styles/colors';
+import {d2, h1, h3} from 'styles/fonts';
+
+export const Background = styled.div`
+  background: rgba(246, 249, 252, 0.36);
+  display: flex;
+  justify-content: center;
+  min-height: 352px;
+  overflow: hidden;
+  position: relative;
+`;
+
+export const Container = styled(SharedContainer)`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  overflow: hidden;
+  width: 100%;
+
+  @media (max-width: 768px) {
+    flex-direction: column;
+  }
+`;
+
+export const LeftContainer = styled.div`
+  margin-right: 48px;
+  padding-left: 48px;
+
+  @media (max-width: 992px) {
+    margin-right: 0;
+  }
+
+  @media (max-width: 768px) {
+    align-self: flex-start;
+    margin-top: 56px;
+    padding: 0 24px;
+  }
+`;
+
+export const TnbText = styled.h1`
+  ${h1.regular};
+  color: #ffcd5c;
+`;
+
+export const Title = styled.h1`
+  ${d2.bold};
+  line-height: 1.25;
+  margin-top: 8px;
+`;
+
+export const SubTitle = styled.h2`
+  ${h3.regular};
+  color: ${colors.palette.gray['500']};
+  line-height: 1.5;
+  margin-top: 16px;
+
+  @media (max-width: 768px) {
+    margin-bottom: 48px;
+  }
+`;
+
+export const RightImage = styled.img`
+  @media (max-width: 992px) {
+    align-self: flex-end;
+    max-width: 400px;
+  }
+
+  @media (max-width: 768px) {
+    bottom: -30px;
+    max-width: 100%;
+    position: relative;
+    right: -40px;
+  }
+`;
+
+export const ImageOuter = styled.img`
+  max-height: 100%;
+  position: absolute;
+  right: 0;
+`;
+
+export const OrangeCircle = styled.div`
+  background-color: #febbbaf0;
```

Comment:
8 chars in a color hex code? 😕 What does the last 2 mean?

Reply:
its for the alpha channel, i.e. opacity: https://stackoverflow.com/questions/6909896/hex-colors-in-android-are-some-times-8-digits-how-what-is-the-difference-betwe

I copied directly from the figma though